content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -5461,10 +5471,18 @@ $connect(): Promise<void>
 
 ### <inlinecode>$on()</inlinecode>
 
+<Admonition type="warning">
+  `$on` is not available in extended clients. Please either migrate to client extensions or use the `$on` method prior to extending your client.
+</Admonition>
+
 The `$on()` method allows you to subscribe to [logging events](#log) or the [exit hook](/concepts/components/prisma-client/working-with-prismaclient/connection-management#exit-hooks).
 
 ### <inlinecode>$use()</inlinecode>
 
+<Admonition type="warning">
+  `$use` is not available in extended clients. Please [either migrate to query extensions](https://www.prisma.io/docs/concepts/components/prisma-client/client-extensions/query) or use the `$use` method prior to extending your client.
```

Comment:
Again: "extended client" does not mean anything to users here. This needs a link to Client extensions docs.